src/sim-host/ui/server/server.js

```diff
@@ -5,21 +5,31 @@ var path = require('path'),
     send = require('send-transform');
 
 module.exports.attach = function (app, dirs, hostRoot) {
-    app.get('/simulator/sim-host.css', function (request, response, next) {
+    app.get('/simulator/sim-host.css', function (request, response) {
         var userAgent = request.headers['user-agent'];
-        if (userAgent.indexOf('Chrome') > -1 && userAgent.indexOf('Edge/') === -1) {
-            next();
-        } else {
-            // If target browser isn't Chrome (user agent contains 'Chrome', but isn't 'Edge'), remove shadow dom stuff from
-            // the CSS file. Also remove any sections marked as Chrome specific.
-            send(request, path.resolve(hostRoot['sim-host'], 'sim-host.css'), {
-                transform: function (stream) {
-                    return stream
-                        .pipe(replaceStream('> ::content >', '>'))
-                        .pipe(replaceStream(/\^|\/shadow\/|\/shadow-deep\/|::shadow|\/deep\/|::content|>>>/g, ' '))
-                        .pipe(replaceStream(/\/\* BEGIN CHROME ONLY \*\/[\s\S]*\/\* END CHROME ONLY \*\//gm, ''));
-                }
-            }).pipe(response);
-        }
+        send(request, path.resolve(hostRoot['sim-host'], 'sim-host.css'), {
+            transform: getTransform(userAgent)
+        }).pipe(response);
     });
 };
+
+function getTransform(userAgent) {
+    if (isChrome(userAgent)) {
+        // If target browser is Chrome, remove any sections marked as not for Chrome.
+        return function (stream) {
+            return stream.pipe(replaceStream(/\/\* BEGIN !CHROME \*\/[\s\S]*\/\* END \*\//gm, ''));
```

Comment:
Worked by coincidence that `replaceStream()` wasn't hitting the second end "tag". Bumped `maxMatchLen` up to 1024 for these matches, and made end "tags" unique.